WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -17,48 +17,55 @@ struct AddEditCoupon: View {
         NavigationView {
             GeometryReader { geometry in
                 ScrollView {
-                    VStack (alignment: .leading) {
+                    VStack (alignment: .leading, spacing: 0) {
                         Group {
```

Comment:
Suggestion: I see a lot of occurrences of `.padding([.bottom], Constants.verticalSpacing)` - what would you think about replacing `Group`s with `VStack`s with the vertical spacing instead?

```suggestion
                        VStack(alignment: .leading, spacing: Constants.verticalSpacing) {
```

Reply:
I believe that it shouldn't work perfectly, because unfortunately when we implemented some reusable views in SwiftUI we didn't take into consideration not using vertical spacing by default.
So this means that some views have a vertical space by default, and other views no.
Eg: `ListHeaderView`, `TitleAndTextFieldRow`, and so on. We always can do a refactor of these reusable views, but it could be hard to discover all the points where they are used, updating the vertical padding everywhere. WDYT?

Reply:
That makes sense! I'm thinking we can get around some of the issues by playing with different VStacks with different spacing, but I understand it can require too much effort for such small details. Let's balance the effort spent then, as long as we can make the view match with the design as closely as possible 😊

Reply:
> That makes sense! I'm thinking we can get around some of the issues by playing with different VStacks with different spacing.

Since these VStacks would integrate different views including dividers, the spacing between the various elements would not match. If we put the dividers outside of the `VStack`, we should still consider the spacing between each `VStack`. It could be tricky, btw this lesson could be a good way to post a P2 with this small suggestion: do not use padding in our reusable components!